packages/app/src/app/pages/Sandbox/Editor/Header/Buttons/RefreshButton.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import UpdateFound from '../UpdateFound';
+import { Action } from './Action';
+
+export const RefreshButton = () => (
+  <Action
+    onClick={() => document.location.reload()}
```

Comment:
@SaraVieira This can just be

```suggestion
    onClick={document.location.reload}
```

Reply:
Nice catch! DOne :)